.circleci/config.yml

```diff
@@ -148,24 +148,24 @@ jobs:
   swift_apitest:
     macos:
       xcode: "12.5.1"
-    working_directory: ~/purchases-ios/APITestersSwiftAPITester
+    working_directory: ~/purchases-ios
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
       - run:
-          name: Build APITesters
-          command: xcodebuild -target SwiftAPITester
+          name: Build SwiftAPITester
+          command: xcodebuild -workspace APITesters/APITesters.xcworkspace -scheme SwiftAPITester
 
   objc_apitest:
     macos:
       xcode: "12.5.1"
-    working_directory: ~/purchases-ios/APITesters.ObjCAPITester
+    working_directory: ~/purchases-ios
```

Comment:
just curious, why is this different from changing the working directory and then running the same command `xcodebuild -workspace APITesters.xcworkspace -scheme SwiftAPITester`?

Reply:
I think it's exactly the same, but also your working directory was `APITesters.ObjCAPITester` and should have been `APITesters` 😄
I prefer to keep my commands from the root of the project, purely just a preference.